core/geometry/src/test/geometry3d/OrderedRotationAngles.test.ts

```diff
@@ -74,46 +64,73 @@ function testMultiAngleEquivalence(ck: Checker, xDegrees: number, yDegrees: numb
     ck.testTightNumber(Angle.degreesToRadians(yDegrees), angles.yRadians, "y radians");
     ck.testTightNumber(Angle.degreesToRadians(zDegrees), angles.zRadians, "Z radians");
 
+    // rotation matrix calculated by classic base rotation matrixes
     const matrixA = angles.toMatrix3d();
-    const matrixB = multipleDegreeRotationsByAxisOrder(-xDegrees, -yDegrees, -zDegrees, orderPair[1]);
-    ck.testMatrix3d(matrixA, matrixB, "Compound rotation pair with order and sign reversal");
```

Comment:
Consider adding a test equivalent to what this deleted test attempted to assert. The intent is to assert that the product matrix M computed from three angles and an AxisOrder (e.g., AxisOrder.XYZ) is the _inverse_ of the product matrix M' computed from the negatives of the angles in the opposite AxisOrder (e.g., AxisOrder.ZYX).  In other words, the assertion is that `I = M*M' = rot(x)*rot(y)*rot(z) * (rot(-z)*rot(-y)*rot(-x))`  The original test asserted something that isn't true (M = M') due to the bug in `multiplyMatricesByAxisOrder`!

Reply:
will address in the next PR